src/cljs/rems/util.cljs

```diff
@@ -76,7 +76,7 @@
   "Fetches data from the given url with optional map of options like #'ajax.core/GET.
 
   Has sensible defaults with error handler, JSON and keywords.
-
+  You can use :simple-error-handler? to decide weather you would like use wrapper for the error handling.
```

Comment:
The confusion about the parameter name is evident in this sentence too, isn't it? This doesn't really say what it does or what the wrapper it's talking about is. One has to read the code.

Reply:
Yes, I agree. The thing is due to so many iterations it is hard to keep the naming clear, there were a lot of discussions about this fix, first from @opqdonut then we had a call with you, then you and Joel discussed it between yourselves ... I will take a look at it with some fresh eyes and try to read through it and then maybe the naming change will be more apparent to me.

Reply:
I think the most important and also the most exciting part of a PR is once you have the solution implemented. Then you refactor (the diff and) the solution until it is actually nice. I.e. "first make it work, then make it right, ...". Only at this stage can you see what the naming actually should be, what is unnecessary, etc.